tsconfig.json

```diff
@@ -6,6 +6,10 @@
     "sourceMap": true,
     "allowJs": true,
     "outDir": "./.build",
+    
+    // Use CJS for better interoperability with legacy modules (e.g. @as-integrations/aws-lambda)
+    "module": "CommonJS",
+    "moduleResolution": "node",
```

Comment:
Hey @alukach, the tests are failing now as the commit this change was a part of, and at a glance this line seems like the most likely culprit. I don't know enough about issue with legacy modules to feel comfortable messing with it, so just wanted to flag it for you.

Reply:
It seems like maybe `"moduleResolution": "node16"` or `"nodenext" should support but ES2020 or CommonJS modules? https://www.typescriptlang.org/tsconfig/#node16nodenext

Reply:
Perhaps related: I just deployed to staging and got the following error when I tried to open the staging frontend:

```
{
    "errorType": "ReferenceError",
    "errorMessage": "exports is not defined in ES module scope\nThis file is being treated as an ES module because it has a '.js' file extension and '/var/task/package.json' contains \"type\": \"module\". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.",
    "stack": [
        "ReferenceError: exports is not defined in ES module scope",
        "This file is being treated as an ES module because it has a '.js' file extension and '/var/task/package.json' contains \"type\": \"module\". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.",
        "    at file:///var/task/src/api/handler.js:5:23",
        "    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)",
        "    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)",
        "    at async /opt/sls-sdk-node/wrapper.js:13440:14",
        "    at async _loadUserApp (file:///var/runtime/index.mjs:1081:16)",
        "    at async UserFunction.js.module.exports.load (file:///var/runtime/index.mjs:1119:21)",
        "    at async start (file:///var/runtime/index.mjs:1282:23)",
        "    at async file:///var/runtime/index.mjs:1288:1"
    ]
}
```